CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+## Set up your development environment
+
+You'll need a Phantom OVA VM image to run this application. To download the image, first register [here](https://my.phantom.us/signup/)
+to create a Phantom account. Phantom will approve your registration and send you credentials for https://my.phantom.us.
+
+Next, log in and go [here](https://my.phantom.us/downloads/) to download the Phantom VM image.
+
+Follow these [instructions](https://docs.splunk.com/Documentation/Phantom/4.10.3/Install/InstallOVA) to install Splunk Phantom 
+as a VM image. Use these [default credentials](https://docs.splunk.com/Documentation/Phantom/4.10.3/Install/Reference) to log in.
+Confirm that you can connect to the VM via SSH on your host machine. You can use `hostname -I` on the remote machine to look up your IP address.
+Also confirm that you can access the Phantom web app at `https://{phantom VM IP address}`.
+
+NOTE: If you are consistently getting timeouts connecting to the Phantom VM, try disconnecting from the VPN and re-attempting.
+
+
+## Deploying the app
+
+Run the below command to install the Code42 app on Phantom. 
+
+```bash
+export PHANTOM_VM_IP_ADDR=0.0.0.0 # Replace with IP address for Phantom VM
+./util.sh move
+```
+
+Open the Phantom web app and login as the `admin` user. Navigate to `Apps > Unconfigured Apps` and find the Code42 App.
+Click Configure New Asset to supply the Console URL, username, and password to connect with. Fill out the fields in Asset Info
+and Asset Settings. Save the Asset Configuration and then click Test Connectivity to test your connection. 
+
+## Creating a virtual environment
```

Comment:
I think some of our other projects have instruction on how to create a venv on windows. Some of our QA folks would probably benefit from that.

Reply:
Creating a virtual environment is only necessary if you want to run unit tests -- I'll add a note to make that more explicit. Unless we're expecting to have some devs switch to Windows I don't think it's necessary to provide virtualenv instructions for Windows.